test/Microsoft.AspNetCore.Routing.Tests/Tree/TreeRouterTest.cs

```diff
@@ -1533,7 +1533,6 @@ public static IEnumerable<object[]> OptionalParamValues
             // Assert
             Assert.NotEqual(nestedValues, context.RouteData.Values);
 
-            // The new routedata is a copy
```

Comment:
This comment was stale, I removed it while I was debugging a failure in this test.